robotoff/app/api.py

```diff
@@ -1055,6 +1061,40 @@ def on_get(self, req: falcon.Request, resp: falcon.Response, username: str):
         resp.media = {"count": {"annotations": annotation_count}}
 
 
+def get_image_list_on_get(req: falcon.Request, resp: falcon.Response):
+    response: JSONType = {}
+    count: int = req.get_param_as_int("count", min_value=1, default=25)
+    page: int = req.get_param_as_int("page", min_value=1, default=1)
+    with_predicted: Optional[int] = req.get_param_as_int("with_predicted", default=1)
```

Comment:
default should be 0.

```suggestion
    with_predicted: Optional[int] = req.get_param_as_int("with_predicted", default=0)
```